src/tablecloth/time/api/converters.clj

```diff
@@ -175,3 +182,14 @@
   (let [^java.time.LocalDate localDate (-> datetime ->local-date)]
     (.with localDate (java.time.temporal.TemporalAdjusters/lastDayOfYear))))
 
+(defn ->year
+  [localdate]
+  (.getYear ^java.time.LocalDate localdate))
+
+(defn ->year-month
+  [localdate]
+  (let [yyyy (.getYear ^java.time.LocalDate localdate)
+        mmm (-> (.getMonth ^java.time.LocalDate localdate)
+                (.getDisplayName ^java.time.format.TextStyle TextStyle/SHORT
+                                 (^java.util.Locale Locale/getDefault)))]
+    [yyyy mmm]))
```

Comment:
@rsiddharthan if we were going to support these types, in order to be consistent with the conversion api these conversion pathways would need to go through the `convert-to` method, which supports universal conversion. i.e. the fn signatured would be `(->year any-datetime)`, and `->year` would be a convenience method on top of `convert-to`. 

However, per our discussion today that we will try *not* supporting the types `Year`, `YearQuarter`, and `YearMonth` these should be removed I think.